solidity/contracts/deposit/Deposit.sol

```diff
@@ -384,6 +391,39 @@ contract Deposit is DepositFactoryAuthority {
     // LIQUIDATION
     //
 
+    /// @notice Get the current collateralization level for this Deposit.
+    /// @dev    This value represents the percentage of the backing BTC value the signers
+    ///         currently must hold as bond.
+    /// @return The severe collateralization level for this deposit.
```

Comment:
@NicholasDotSol potential copypasta re: "severe"?

Reply:
Also is there a reason why `getCollateralizationPercentage` is uint256? Seems like we could return uint16 here too, just curious.

Reply:
> @NicholasDotSol potential copypasta re: "severe"?

yeah good catch, severe -> current

> Also is there a reason why getCollateralizationPercentage is uint256? Seems like we could return uint16 here too, just curious.

There is no reason to move this to a smaller uint size. It makes sense for other collateralization values that take up storage since we can take advantage of bit-packing. But Since this value is calculated in memory, converting from and to uint types will just take additional gas

Reply:
> yeah good catch, severe -> current

also note that this is repeated below too. 

> There is no reason to move this to a smaller uint size. It makes sense for other collateralization values that take up storage since we can take advantage of bit-packing. But Since this value is calculated in memory, converting from and to uint types will just take additional gas

Alright, fair enough. Didn't know that there were gas costs for casting to lower types, thought everything was the same word-size in memory.